source/CINetworkMessageManager.cpp

```diff
@@ -172,8 +175,10 @@ void NetworkMessageManager::receiveMessages() {
                 int timestamp = NetworkUtils::decodeInt(recv[16], recv[17], recv[18], recv[19]);
                 
                 CULog("RCVD PU> SRC[%i], CLR[%i], SIZE[%f]", srcPlayer, planetColor, planetSize);
-
-                std::shared_ptr<OpponentPlanet> planet = OpponentPlanet::alloc(0, 0, static_cast<CIColor::Value>(planetColor));
+                
+                CILocation::Value corner = NetworkUtils::getStardustLocation(getPlayerId(), srcPlayer);
```

Comment:
NIT: might be better to handle the corner somewhere else (maybe game update manager) so we don't need to pass extra fields in here